app/models/order.rb

```diff
@@ -7,4 +7,13 @@ class Order < ApplicationRecord
   has_one :online_code, dependent: :destroy
   belongs_to :employee
   belongs_to :reward
+
+  enum delivery_type: { post: 'post', pickup: 'pickup' }
+
+  def type_and_address
+    return 'Online' if reward_snapshot.online?
+    return 'Pickup' if pickup?
```

Comment:
I would consider adding 'delivery' suffix to this enum - so we have pickup_delivery and post_delviery helpers